lib/src/message_handlers.dart

```diff
@@ -112,19 +113,21 @@ class NavigationMessageHandler implements MessageHandler {
 class NavigationStopMessageHandler implements MessageHandler {
   @override
   void handleMessage(
-      SitumFlutterWYF situmFlutterWYF, Map<String, dynamic> payload) {
-    var sdk = SitumFlutterSDK();
+      MapViewController situmFlutterWYF, Map<String, dynamic> payload) {
+    var sdk = SitumSdk();
     sdk.stopNavigation();
   }
 }
 
 class PoiSelectedMessageHandler implements MessageHandler {
   @override
   void handleMessage(
-      SitumFlutterWYF situmFlutterWYF, Map<String, dynamic> payload) async {
-    var poiId = "${payload["identifier"]}";
+      MapViewController situmFlutterWYF, Map<String, dynamic> payload) async {
     if (situmFlutterWYF._onPoiSelectedCallback != null) {
-      var poi = await situmFlutterWYF._fetchPoiFromCurrentBuilding(poiId);
+      var poiId = "${payload["identifier"]}";
```

Comment:
Xa que cambiamos a buildingIndentifier, case molaba que a variable non usase os Id en lugar de Identifier, para que quede todo máis coherente

Reply:
E se na do poi, chegase poiIdentifier en lugar de identifier, xa quedaba todo supercoherente, pero bueno, non pasa nada